voucher-mission/src/main/java/org/prgrms/vouchermission/FixedAmountVoucher.java

```diff
@@ -24,16 +26,21 @@ public UUID getVoucherId() {
 
     @Override
     public long discountAmount(long beforeDiscount) {
-        return amount;
+        return beforeDiscount - amount;
     }
 
-    @Override
+    public long getAmount() { return amount;}
```

Comment:
```suggestion
    public long getAmount() { 
        return amount;
    }
```
비슷한 역할을 하는 코드는 일관성 있는 style을 유지하면 가독성이 높아요

Reply:
수정하겠습니다!